geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseDataStore.scala

```diff
@@ -68,6 +68,10 @@ class HBaseDataStore(con: ConnectionWrapper, override val config: HBaseDataStore
 
   override def dispose(): Unit = try { super.dispose() } finally { con.close() }
 
+  override def delete():Unit={
+    super.delete()
+    metadata.asInstanceOf[HBaseBackedMetadata[String]].resetCache()
```

Comment:
this is going to affect all data store impls - could we remove this change here, and put it in `GeoMesaDataStore.delete`?

Reply:
> this is going to affect all data store impls - could we remove this change here, and put it in `GeoMesaDataStore.delete`?

I didn't catch you, why HBase impls is going to affect all impls? or what you mean is to affect all implementations so it should be moved to geomesadata?

Reply:
right, sorry, the bug is going to affect all the data stores so I'd like to fix it generally instead of just for hbase